templates/upstreams.j2

```diff
@@ -7,6 +7,8 @@
 {% set _ = data.update({config_block.proxy_pass: "1"}) %}
   upstream {{ config_block.proxy_pass }} {
 {% if 'proxy_pass' in config_block  %}
+{% if 'no_upstream' in config_block and config_block.no_upstream == 'true' %}
```

Comment:
do you really need empty section here? if not - try `{% if 'no_upstream' not in config_block and config_block.no_upstream == 'true' %}`

Reply:
^ that won't work. I could do `{% if 'no_upstream' not in config_block and ('no_upstream' in config_block and config_block.no_upstream == 'true') %}` but i think it makes it less readable